HW1/P2/P2a.py

```diff
@@ -1,3 +1,23 @@
 from P2 import *
 
 # Your code here
+
+
+# make pixels
+i = sc.parallelize(range(1000))
+j = sc.parallelize(range(1000))
+
+pxls = i.cartesian(j)
+
+
+
+# put through mandelbrot func
+result = pxls.map(lambda x: (x , mandelbrot(x[0]/500.0 - 2,x[1]/500.0 - 2)))
+
+
+
+#draw_image(result)
+plt.hist(sum_values_for_partitions(result).collect())
+plt.savefig('P2a_hist2.png')
+
+print "got here"
```

Comment:
Nice and clean solution for P2a with i, j and pxls. Mind the range for mandelbrot was supposed to be 2000 by 2000 pixels.